README.md

```diff
@@ -181,6 +181,45 @@ ENV BROKER_CLIENT_URL   http://my.broker.client:8000
 ENV PORT                8000
 ```
 
+### Azure Repos
+
+To use the Broker client with [Azure](https://azure.microsoft.com/en-us/services/devops/), run `docker pull snyk/broker:azure-repos` tag. The following environment variables are mandatory to configure the Broker client:
+
+- `BROKER_TOKEN` - the Snyk Broker token, obtained from your Azure Repos integration settings view (app.snyk.io).
+- `AZURE_REPOS_TOKEN` - an Azure Repos personal access token
+- `AZURE_REPOS_ORG` - organization name, could be found in Organization Overview page
+- `PORT` - the local port at which the Broker client accepts connections. Default is 7341.
+- `BROKER_CLIENT_URL` - the full URL of the Broker client as it will be accessible by Azure Repos webhooks, such as `http://my.broker.client:7341`
```

Comment:
```suggestion
- `BROKER_CLIENT_URL` - the full URL of the Broker client as it will be accessible by your Azure Repos' webhooks, such as `http://my.broker.client:7341`
```